util/dvsim/Deploy.py

```diff
@@ -463,6 +465,7 @@ def pre_launch(self):
         self.launcher.renew_odir = True
 
     def post_finish(self, status):
+        self.job_runtime, self.simulated_time = get_runtimes(self.get_log_path(), self.sim_cfg.tool)
```

Comment:
This needs to gracefully handle the exception if tool != vcs/xcelium. Otherwise, dvsim will exit throwing an assertion error after successfully completing the sim.

For other tools, `job_runtime` can be 'loosely' computed and returned by dvsim. @matutem is adding `job_timeout` mechanism for the `Launcher` class, which can also return the `job_runtime` as maintained by dvsim (which will of course, be less accurate that what the simulator reports).